internal/versions/versions_test.go

```diff
@@ -743,10 +771,72 @@ var _ = Describe("list versions", func() {
 	})
 })
 
+var _ = FDescribe("test", func() {
+	var (
```

Comment:
there are several issues to fix here, please see the diff here: https://gist.github.com/nmagnezi/623f2902fc37e4d0d90572f5abaa194f

with that, tests are passing for me locally.